hudi-common/src/main/java/org/apache/hudi/common/table/view/TableFileSystemView.java

```diff
@@ -149,10 +149,12 @@ interface SliceView extends SliceViewWithLatestSlice {
    */
   Stream<HoodieFileGroup> getAllFileGroups(String partitionPath);
 
+  Stream<Pair<String, List<HoodieFileGroup>>> getAllFileGroups(List<String> partitionPaths);
```

Comment:
We can do away with this api, doesn't add much value. Instead, we can directly use the `getAllFileGroups(String partitionPath)` api at the call site.